lib/quoters/WebhookQuoter.ts

```diff
@@ -83,9 +83,6 @@ export class WebhookQuoter implements Quoter {
       const opposingCleanRequest = request.toOpposingCleanJSON();
       opposingCleanRequest.quoteId = uuidv4();
 
-      this.log.info({ request: cleanRequest, headers }, `Webhook request to: ${endpoint}`);
-      this.log.info({ request: opposingCleanRequest, headers }, `Webhook request to: ${endpoint}`);
```

Comment:
oh wait I use these logs to debug quoter integrations.

Reply:
They are decent sanity check to make sure I know what the requests look like that we are sending them.

Reply:
we also have this log 
```        
    ......,
    {
            "endpoint": "https://rfq.inupaws.com/",
            "name": "Inu Paws",
            "addresses": [
                "0x6af49606d941cda32de1ee94397dc821ceb09dac"
            ]
        },
        {
            "endpoint": "https://uniswap.barterswap.xyz/solveUniswapX",
            "name": "Barter",
            "addresses": [
                "0xbbf8e7cc7a29af9eb149beb9d83cfe015719f9ef"
            ]
        }
    ],
    "msg": "Fetching quotes from 9 endpoints",
```
happy to retain them if both are helpful to you

Reply:
Yeah i think both are because these ones show the full request so I can query and see what went wrong. At least for now. Maybe a TODO to remove at some point.